longest_common_subsequence/myers.py

```diff
@@ -0,0 +1,151 @@
+from math import ceil
+
+class EditBox:
+  def __init__(self, text_a, text_b, a_start, a_end, b_start, b_end):
+    self.text_a = text_a
+    self.text_b = text_b
+    self.a_start = a_start
+    self.a_end = a_end
+    self.b_start = b_start
+    self.b_end = b_end
```

Comment:
I guess you may make multiple assignments by pairs or even like that:
self.text_b, self.b_start, self.b_end = text_b, b_start, b_end